nodeup/pkg/model/docker.go

```diff
@@ -188,7 +188,7 @@ var dockerVersions = []packageVersion{
 				Hash:    "a6b0243af348140236ed96f2e902b259c590eefa",
 			},
 		},
-		Dependencies: []string{"libtool-ltdl", "libseccomp"},
+		Dependencies: []string{"libtool-ltdl"},
```

Comment:
These are the ones we're moving into packages.go?

Reply:
Yes. Moving to packages.go.